providers/curation/github.js

```diff
@@ -227,17 +232,15 @@ class GitHubCurationService {
   }
 
   async ensureCurations() {
-    if (this.curationUpdateTime && (Date.now - this.curationUpdateTime < this.options.curationFreshness))
-      return;
+    if (this.curationUpdateTime && Date.now - this.curationUpdateTime < this.options.curationFreshness) return;
```

Comment:
getting a little long. One could argue that in the interest of clarity it should really be 

```javascript
  const cacheStillValid = this.curationUpdateTime && Date.now - this.curationUpdateTime < this.options.curationFreshness;
  if (cacheStillValid) return;
```
or
```javascript
    const age = this.curationUpdateTime ? Date.now - this.curationUpdateTime : 0;
    if (age < this.options.curationFreshness) return;
```
and that that code would actually be clearer. I could go with that.

Reply:
+1 for readability. When someone reads that code for the first time, i.e. `if (cacheStillValid) return;` they immediately know what is going on.